fluid-project/handbook.floeproject.org
# fix: add missing hover and focus styles

## Description

This update adds remaining hover and focus styles specified in the design which were not implemented for the `v1.0.0` release.

## Steps to test

1. Go to the Approaches page
2. Hover over or focus on each of the following elements on the page:
   - the "An Inclusive Design Research Centre project" link above the page header
   - the breadcrumb link to "Home"
   - the "expand all" and "collapse all" buttons
   - each subsection accordion control
3. Click on "Good Practices" to expand it
4. Click on "Follow Accessibility Principles"
5. Once the page is loaded, hover over or focus on the links within the page content

**Expected behavior:** Each of the elements specified above should have a hover/focus indicator which does not look broken.

## Additional information

One of the CSS classes added in this update makes use of the `.topic` selector which is not in the `dev` branch at the time of filing. This will be addressed once #85 is merged into `dev`, at which point this PR will be marked ready for review & merger.

The breadcrumb focus/hover indicators were not specified in the design but were added to improve accessibility.

## Related issues

Resolves #61
